cypress/integration/api/magic.test.ts

```diff
@@ -1,14 +1,16 @@
+import { HttpMethod } from "@lib/http-method";
 import { StatusCodes } from "http-status-codes";
 
-describe("test magic api", () => {
-  it("should give a 405 when trying to GET", () => {
+describe("test send magic link api", () => {
```

Comment:
This test still fails for me, even with the sendgrid API key installed (I get a 500 from the endpoint)

Reply:
What's the error message?

Reply:
I forgot to remove the comment after the API key, nvm.